### DISCUSSION TITLE
Singleton Collection Import Export Support

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#22603] [Feature Requests]
### Summary

Since we can import/export other collection but not singleton collections, this is a huge downer when using web interface.

### Motivation

Importing and Exporting via Web interface

### Detailed Design

Same as for collections.

### Must Have:
- Import & Export Button

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by br41nslug** [comment]
I know this doesnt address the nice UI or import part of this but you can currently achieve the export as workaround by using a button interface that opens `/items/<singleton collection>?export=json` (for singletons specifically since it is only one item)

**Comment by NilsBaumgartner1994** [comment]
Oh that’s nice :-)Beste Grüße,Nils BaumgartnerAm 29.05.2024 um 20:24 schrieb Brainslug ***@***.***>:﻿
I know this doesnt address the nice UI part of this but you can currently achieve this as workaround by using a button interface that opens /items/<singleton collection>?export=json (for singletons specifically since it is only one item)
